netlify.toml

```
@@ -6,7 +6,7 @@
   package = "@netlify/plugin-lighthouse"
 
   [plugins.inputs.thresholds]
-    performance = 0.77
+    performance = 0.60
```

```suggestion
    performance = 0.75
```

Increase it to 0.75 since [the latest successful build managed to achieve that score](https://app.netlify.com/sites/wargabantuwarga/deploys/60eb0682294a310008e7696d?utm_source=github&utm_campaign=bot_dl)